cassandra/src/test/java/org/hibernate/ogm/datastore/cassandra/test/id/GuitarPlayer.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.cassandra.test.id;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.TableGenerator;
+
+/**
+ * @author Nicola Ferraro
+ */
+@Entity
+public class GuitarPlayer {
```

Comment:
Could you think about whether we can re-use these types across different dialects?